php/geoimagesearch.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+//This file is autogenerated. See modules.json and autogenerator.py for details
+
+/*
+    geoimagesearch.php
+
+    MediaWiki API Demos
+    Demo of `Geosearch` module: Use generator module
+	to get search results for pages near Wikimedia HQ
+	with images
+
+    MIT License
+*/
+
+$endPoint = "https://en.wikipedia.org/w/api.php";
+$params = [
+    "action" => "query",
+    "generator" => "geosearch",
+    "prop" => "coordinates|pageimages",
+    "ggscoord" => "37.7891838|-122.4033522",
+    "format" => "json"
+];
+
+$url = $endPoint . "?" . http_build_query( $params );
+
+$ch = curl_init( $url );
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+$output = curl_exec( $ch );
+curl_close( $ch );
+
+$result = json_decode( $output, true );
+var_dump( $result );
```

Comment:
Here we obtain more image details like in: https://github.com/wikimedia/mediawiki-api-demos/blob/5d6745ecd1cb61dcbbdb5b96547329e589dff390/python/geoimagesearch.py#L35